src/AppContext.js

```diff
@@ -17,12 +18,28 @@ const AppProvider = props => {
     moduleData: props.moduleData,
   });
 
+  const filterPastEvents = eventList => {
+    let eventArray = [];
+    let currentDate = dayjs(new Date());
+
+    eventList.forEach(e => {
```

Comment:
Instead of iterating over `eventList` and pushing items into a holder array, you could use [`.filter()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter)
```javascript
eventList.filter(e =>
  eventStartDate.isAfter(currentDate)
)
```